packages/sandbox/ui/src/components/component-controls/ValueCheckbox.tsx

```diff
@@ -1,19 +1,32 @@
 import React from 'react'
-import { startWithType, mapHandlers, pureComponent, mapState, mapDebouncedHandlerTimeout } from 'refun'
+import { startWithType, mapHandlers, pureComponent, mapState, mapDebouncedHandlerTimeout, mapWithProps } from 'refun'
 import { Switch } from '../switch'
 import { SYMBOL_SWITCH } from '../../symbols'
 
 export type TValueCheckboxProps = {
   propPath: readonly string[],
   checkedPropValue: any,
   propValue: any,
+  propPossibleValues?: readonly any[],
   onChange: (propPath: readonly string[], propValue: any) => void,
 }
 
 const isDefined = (val: any): boolean => val !== false && val !== undefined
 
+const hasFalseInPosiibleValues = (propPossibleValues: readonly any[]): boolean => propPossibleValues.length === 2 && propPossibleValues.includes(true) && propPossibleValues.includes(false)
```

Comment:
Maybe the name is a bit misleading? `hasFalseInPosiibleValues` sounds like we should support something like this:
```
isChecked: [false],
```
while the code suggest that we only support `false` if it's together with `true` in an array.
How about `hasAcceptableFormat` or `isInSupportedFormat`?

Reply:
Or `hasFalseInPosiibleValuesAndIsInSupportedFormat`? 😅

Reply:
`hasBothTrueAndFalseValues`? `isPossibleValuesAreTrueAndFalse`? oooof, naming is crazy

Reply:
I like `hasBothTrueAndFalseValues` 🙂 It's super clear

Reply:
switch the names 🔄 😌